.github/workflows/nightly-throughput-stress.yml

```diff
@@ -25,6 +25,9 @@ on:
         default: 360
         type: number
 
+permissions:
+  contents: read
```

Comment:
FYI a slightly more complete `permissions` block is being added in https://github.com/temporalio/sdk-typescript/pull/1815